### ISSUE TITLE
Add support for ueberzug

### ISSUE THREAD

**Issue by colaxdev** [helix-editor/helix#2345]
# Idea

[Ueberzug](https://github.com/seebye/ueberzug) is a tool for show images in terminal. [Telescope](https://github.com/nvim-telescope/telescope-media-files.nvim) already use this:

![](https://camo.githubusercontent.com/92dfc650e349ac5bb936f9cdfeac85e132d9fb87bfde8f289fefe6520beded6c/68747470733a2f2f692e696d6775722e636f6d2f77454f3034544b2e676966)

**Comment by kirawi** [comment]
There's https://github.com/atanunq/viuer which would allow this feature, but I'm not sure that this is something that belongs as a core feature of Helix. Maybe as a plugin instead?

**Comment by Mathijs-Bakker** [comment]
Note to consider: Ueberzug is restricted to linux only.

**Comment by colaxdev** [comment]
Yes but if there is a plugin Helix should have a plugin system...

**Comment by kirawi** [comment]
There's a tracking issue for a Wasm-based plugin system: #122
